src/api/graphql-client.ts

```diff
@@ -168,13 +169,14 @@ export const me = async (graphqlApiUrl: string, accessToken: string): Promise<Ex
         user {
           id
           username
+          avatar
           email
           walletAddress
         }
       }
     `,
     variables: { data: { accessToken } },
-    dataField: 'me',
+    dataField: 'user',
```

Comment:
At the moment the dataField's set as the `operationName`, for any of the remaining queries. Are you sure you need to change it?

```suggestion
    dataField: 'me',
```

Reply:
That may work on the project function because the server operation name is `project` and the server returns the data under the same name `project`. Therefore, it makes sense to make both the `operationName` and `dataField` be `project`.

The same thing doesn't happen in the `me` function. Notice that, although the operation name is `me` the returned data actually is received from the BE under the prop `user` therefore this change is needed unless something changes on the server side.

Am I missing something?

Reply:
@bcamarneiro Yeh, seems like a redundancy. We can remove it to avoid confusion see https://github.com/fleek-platform/login-button/pull/25

Reply:
@bcamarneiro removed, you can pull latest now.